styx-api-service/src/test/java/com/spotify/styx/api/WorkflowResourceTest.java

```diff
@@ -382,6 +426,26 @@ public void shouldReturnWorkflowInstanceData() throws Exception {
     assertJson(response, "triggers.[0].executions.[0].statuses.[1].timestamp", is("2016-08-10T07:00:02Z"));
   }
 
+  @Test
+  public void shouldPaginateWorkflowInstancesData() throws Exception {
```

Comment:
Could you add a test where the `offset` parameter is set as well? I kind of get the feeling that there could be an off-by-one error in there.

```java
scan.setStartRow(Bytes.toBytes(offsetInstance.toKey() + '#'));
```